### ISSUE TITLE
📚 Documentation: Buttons (links) on docs homepage do not have focus indicators [a11y][TheA11y100]

### ISSUE THREAD

**Issue by GrahamTheDevRel** [appwrite/website#283]
## Issue
The buttons (links) for different technologies on the docs homepage do not have visible focus indicators.

![Flutter link is focused, shown via DevTools with "focus", "focus-within" and "focus-visible" yet no focus indicator appears](https://github.com/appwrite/website/assets/121672086/5ed54f08-833e-42ff-998d-dc3d6cf9f863)

## Why it matters
People navigating via keyboard rely on focus indicators to make sure they do not get lost on the page.

## Suggested fix
Add focus indicators. They should have at least a 3:1 contrast ratio.

## Relevant Success Criterion WCAG
[WCAG focus visible](https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html)
[WCAG focus appearance](https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance.html)

## Additional Notes
[Issue 276](https://github.com/appwrite/website/issues/276) has suggestions for a focus indicator colour that could work, but please check with Appwrite before proceeding. 

@jasonetorres was interested in fixing this one.

[Found on stream on 2023-10-25](https://www.twitch.tv/videos/1959992199)

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by jasonetorres** [comment]
I'll take it!

**Comment by TorstenDittmann** [comment]
This needs some insight from the design team before I can assign someone 👍🏻

**Comment by jasonetorres** [comment]
@TorstenDittmann yeah @GrahamTheDevRel did mention there were a couple similar cases. I'd love to be involved so just lmk!

**Comment by eldadfux** [comment]
Starting today this repo is archived. Please raise any issues with the new Appwrite website on the main platform repo: https://github.com/appwrite/appwrite
